.github/workflows.src/tests.inc.yml

```diff
@@ -180,8 +181,9 @@
           rm -rf ${BUILD_LIB}
           mkdir -p ./build/extensions
           rsync -aP ./build/extensions/ ${BUILD_LIB}/
-          python setup.py -v build_ext
+          BUILD_EXT_MODE=py-only python setup.py -v build_ext
           rsync -aP ${BUILD_LIB}/ ./build/extensions/
```

Comment:
Let's use `-v` instead of `-P`, because the latter actually implies `--partial` (and we don't want that), and also, `--progress` output gets garbled in the logs.